posthog/constants.py

```diff
@@ -216,9 +216,14 @@ class AnalyticsDBMS(str, Enum):
     CLICKHOUSE = "clickhouse"
 
 
+UNIQUE_USERS = "dau"
 WEEKLY_ACTIVE = "weekly_active"
 MONTHLY_ACTIVE = "monthly_active"
 
+# Special math_property value indicating average event count per user instead of an event property
+# Two underscored signify that this is not any property
+EVENT_COUNT_PER_ACTOR = "__event_count_per_actor"
```

Comment:
This is only "per user" currently, but we should be able to easily extend this to groups in a future PR. In that case, `math_property` will be the same, it's just that `math_group_type_index` would also be set.

Reply:
Follow-up!